packages/charts/src/chart_types/flame_chart/flame_chart.tsx

```diff
@@ -1160,7 +1161,8 @@ class FlameComponent extends React.Component<FlameProps> {
         </div>
         <BasicTooltip
           canPinTooltip={canPinTooltip}
-          onPointerMove={() => ({ type: ON_POINTER_MOVE, position: { x: NaN, y: NaN }, time: NaN })}
+          // @ts-ignore redux-toolkit complains about this
+          onPointerMove={() => onPointerMove({ position: { x: NaN, y: NaN }, time: NaN })}
```

Comment:
```suggestion
          onPointerMove={() => onPointerMove({ position: { x: NaN, y: NaN }, time: NaN })}
```

I'm not sure why this was typed this way...

```ts
interface TooltipDispatchProps {
  onPointerMove: typeof onPointerMoveAction
  // ...
}
```

 but I think it should be...

```ts
interface TooltipDispatchProps {
  onPointerMove: (...args: Parameters<typeof onPointerMoveAction>) => void;
  // ...
}
```

The `tooltip.tsx` file does not require the return value of the action once passed through `dispatch`.

Reply:
Thanks, fixed it in [869226c](https://github.com/elastic/elastic-charts/pull/2609/commits/869226c0e78d1592061264f00885d677a9565aa6).